src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
+    }
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
Method hide should change hidden property to the opposite value, not always to the true.

Reply:
[FUNCTION NAMING] - if the function has a name hide this function should just hide but not toggle

Reply:
Good comment 👍 Truly checklist disagrees with the task description.
I will keep that in mind 👌